lang/zh-CN.lng

```diff
@@ -121,40 +121,40 @@ ConfigurationTabControl.MainTabPage=常规
 InstallOptionsGroup=安装选项
 PartSchemeLabel=分区表：
 ImgInstallLabel=安装镜像为：
-EnableLZXCheck=启用 LZX 压缩，可以减小占用空间。
+EnableLZXCheck=启用 LZX 安装压缩
 BootOptionsGroup=启动选项
-MessageBox.BootConfigEditorWarn=修改默认启动配置可能会导致设备无法启动。{n}{n}是否继续？
+MessageBox.BootConfigEditorWarn=修改默认的启动配置可能会导致出现故障甚至根本无法启动。{n}{n}是否仍然继续？
 ConfigurationTabControl.AdvancedTabPage=高级
-MessageBox.AdvancedTabPageWarn=在高级设置菜单中更改默认设置可能会禁用部分设备功能或导致系统无法启动。{n}{n}你确定要继续吗？
+MessageBox.AdvancedTabPageWarn=在高级设置菜单中更改默认设置可能会禁用特定设备功能或导致系统无法启动。{n}{n}您确定要继续吗？
 WindowsBootOptionsGroup=Windows 启动选项
 MemoryLimitLabel=内存限制：
 RecoveryEnabledCheck=启用 Windows 恢复环境
 WoROptionsGroup={appname} 选项
-SaveConfigOnExitCheck=退出时保存这次所用配置。
-SaveConfigOnExitCheckInfoLabel=提醒: 未开启的选项（以及{0}菜单中选项）将不会被保存。
+SaveConfigOnExitCheck=退出时保存此配置
+SaveConfigOnExitCheckInfoLabel=提醒: 已禁用的选项（以及{0}菜单中选项）将不会被保存。
 
 [SetupOverviewPage]
 PageTitle=安装
 InstallOverviewLabel=安装概览
-InstallWarnBox=警告，您的设备数据将会被清除！请提前对设备数据做好备份！
+InstallWarnBox=警告，您设备的所有数据都将会被清除！请提前对您的文件做好备份！
 OSLabel=要安装的操作系统
 DriversPathLabel=驱动程序路径
-UEFIPathLabel=UEFI固件路径
+UEFIPathLabel=UEFI 固件路径
 PartitionSchemeLabel=所使用的分区表
-ImageDeployerLabel=释放镜像方式
-LZXCompressionLabel=是否启用 LZX 压缩
-LZXCompression.YES=是
-LZXCompression.NO=否
+ImageDeployerLabel=部署镜像方式
+LZXCompressionLabel=LZX 安装压缩
+LZXCompression.YES=已启用
+LZXCompression.NO=已禁用
 
 [SetupProgressPage]
 PageTitle=安装
-InstallInfoBox=此过程可能需要很长时间，具体取决于你的SD卡性能。
-TryAgainBtn=重试一遍
-InstallStatusLabel.GettingReady=准备就绪
+InstallInfoBox=此过程可能需要很长时间，具体取决于您的SD卡性能。
```

Comment:
Can you replace "SD" with "device" like in the English translation? https://github.com/worproject/WoR-Translations/blob/dc56b2b88ff664f18dfc9c5d1dfb027750c65071/lang/en-US.lng#L151

Reply:
sure!